project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+      - &rim_joist_insulation
+        option: Insulation Rim Joist|R-10, Exterior
+        apply_logic: 
+          - and:
+            - or:
+              - Geometry Foundation Type|Unvented Crawlspace
+              - Geometry Foundation Type|Vented Crawlspace
+              - Geometry Foundation Type|Heated Basement
+            - Insulation Foundation Wall|Uninsulated
+        costs:
+          - value: 2.32 # $/ft2 (1.8* $/ft2 * 1.289 inflation) https://remdb.nrel.gov/measures.php?gId=14&ctId=378&scId=5929&acId=5932)
+            multiplier: Rim Joist Area, Above-Grade, Exterior (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #seal crawlspace vents
+      - &seal_vented_crawlspaces
+        option: Geometry Foundation Type|Unvented Crawlspace
+        apply_logic:
+          and:
+            - Geometry Foundation Type|Vented Crawlspace
+            - or: # Ensure that the dwelling unit being modeled is either (1) not in a multifamily building, or (2) on the bottom floor of a multifamily building and therefore above the crawlspace.
+                - Geometry Building Level MF|None
+                - Geometry Building Level MF|Bottom
+        costs:
+          - value: 1.083 # $/ft2 (0.84* $/ft2 * 1.289 inflation) https://remdb.nrel.gov/measures.php?gId=14&ctId=334&scId=5169&acId=6565)
+            multiplier: Floor Area, Foundation (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #insulate finished attics and cathedral ceilings to R-30
+  - upgrade_name: Advanced Enclosure
+    #Up
+    options:
+      #Light Enclosure Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *infiltration_30pct_reduction
+      #Intermediate Enclosure Measures
+      - *ducts_category1
+      - *ducts_category2
+      - *ducts_category3
+      - *ducts_category4
+      #- *ducts_category5
+      #- *ducts_category6
+      - *ducts_category7
+      - *Drill-and-fill
+      - *foundation_wall_insulation
+      - *foundation_wall_insulation_finished_basement
+      - *rim_joist_insulation
+      - *seal_vented_crawlspaces
+      #EnergyStar Windows
+      - *EnergyStar_window_IECC_CZ1A-2A-2B
+      - *EnergyStar_window_IECC_CZ3A-3B-3C
+      - *EnergyStar_window_IECC_CZ4A-4B-4C
+      - *EnergyStar_window_IECC_CZ5A-5B-6A-6B-7A-7B
+      #1” (TBD EPS or XPS) Exterior wall insulation for dwelling units with wall insulation of less than R-19  
+      - &exterior_wall_insulation
+        option: Insulation Wall|ICF, 2-in EPS, 4-in Concrete, 2-in EPS #(TBD)
```

Comment:
I don't think this is the correct way to apply external insulation. We'll need to make one for every existing option. e.g., Wood Stud, R-7 >> Wood Stud, R-7, 2-in EPS

Reply:
Also, I think this needs to be consolidated with "drill-and-fill" since they both tough Insulation Wall.

Reply:
Are there any reference that I can get the wall_assembly_r for Wood Stud, R-7, 2-in EPS?
"drill-and-fill" only apply for homes with wood stud walls and no insulation. How about exclude Wood Stud, Uninsulated from the option exterior_wall_insulation? In that way, there will not have overlap.

Reply:
add Insulation Sheathing|R-5|ResStockArguments|wall_continuous_exterior_r=5	to represent adding 1” exterior insulation XPS (R-5/in) for walls